src/components/CreatePostInput/index.js

```diff
@@ -105,19 +105,6 @@
     cancelTokenRef.current = axios.CancelToken.source();
   };
 
-  const isTopicInDeletedText = (event) => {
-    const {nativeEvent} = event;
-    const isBackspacePressed = nativeEvent?.key === 'Backspace';
-    const deleteHashtagDetected =
-      lastTopicInCursor !== '' && lastTopicInCursor.indexOf(' ') === -1 && isBackspacePressed;
-    if (!deleteHashtagDetected) return;
-
-    const newTopics = [...topics];
-
-    newTopics.splice(newTopics?.lastIndexOf(lastTopicInCursor), 1);
-    setTopics(newTopics);
-  };
-
   const handlePostTextChanged = (v) => {
     setMessage(v);
     // updateHashtag(v, topics, setTopics)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `updateHashtag(v, topics, setTopics)` has been commented out. If this function is no longer needed, consider removing it entirely to avoid confusion. If it might be needed in the future, add a comment explaining why it's commented out.
```diff
-      // updateHashtag(v, topics, setTopics)
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->